v2/i18n/localizer_test.go

```diff
@@ -577,6 +577,15 @@ func localizerTests() []localizerTest {
 			},
 			expectedErr: &MessageNotFoundErr{tag: language.AmericanEnglish, messageID: "Hello"},
 		},
+		{
+			name:            "empty default message",
+			defaultLanguage: language.English,
+			acceptLangs:     []string{"en"},
+			conf: &LocalizeConfig{
+				DefaultMessage: &Message{},
+			},
+			expectedErr: &MessageNotFoundErr{tag: language.English, messageID: ""},
```

Comment:
Can you update the test to use a non-empty message ID?

Reply:
Sure but that won't be exactly what mattermost is testing for. I don't know if that can matter in case of a code revamp

Reply:
Could add two test cases if you want